deploy.go

```diff
@@ -182,6 +182,16 @@ func (p *deploycmd) deployAll(c *cli.Context, appName string, appf *appfile) err
 				ff.Path = p2
 			}
 		}
+		if appf.Config != nil {
```

Comment:
Isn't this a piece of repeated code?

Reply:
What do you mean from where?

Reply:
The next line is checking func file config this is checking app file config. If we have no app file config which is appf then we do not need to add config to the func file. And the next one is checking if there is a func file config and if there is not initializing it to a map since we cannot assign values into a nil map.